app/lib/frontend/handlers/misc.dart

```diff
@@ -67,6 +73,25 @@ Future<shelf.Response> readinessCheckHandler(shelf.Request request) async {
   }
 }
 
+/// Handles requests for /topics
+Future<shelf.Response> topicsPageHandler(shelf.Request request) async {
+  late Map<String, int> topics;
+  try {
+    final data = await storageService
+        .bucket(activeConfiguration.reportsBucketName!)
+        .readAsBytes(topicsJsonFileName);
+
+    topics = utf8JsonDecoder.convert(data) as Map<String, int>;
+  } on FormatException {
+    topics = {};
+  } on DetailedApiRequestError catch (e, st) {
+    if (e.status == 404) {
+      topics = {};
+    }
+  }
```

Comment:
```suggestion
  } on FormatException {
    topics = {};
    // TODO: Log error loading topics
  } on DetailedApiRequestError catch (e, st) {
    topics = {};
    if (e.status == 404) {
      // TODO: Log inability to load topics
    }
  }
```

Reply:
We can do the TODO later, I don't think that's particularly blocking.

If we want it we just need:
```dart
import 'package:logging/logging.dart' show Logger;

final _log = Logger('pub.handlers.misc');
```
at the top of the file.

Reply:
added loggin.